addon/services/ember-oauth2.js

```diff
@@ -252,8 +254,8 @@ export default Ember.Service.extend(Ember.Evented, {
    * @method authUri
    * @return {String} Authorization uri for generating an OAuth2 token
    */
-  authUri: function() {
-    var uri = this.get('authBaseUri');
+  authUri() {
+    let uri = this.get('authBaseUri');
```

Comment:
can be `const`

Reply:
This can't be a `const` variable since it is reassigned later at

```js
uri +=
  '?response_type=' +
  ...
```